random-menu-generator.rb

```diff
@@ -0,0 +1,27 @@
+# RANDOM FOOD GENERATOR
+
+puts "Please enjoy one of our following specials:"
+# Three arrays to hold food lists
+adj_one = ["healthy", "cold", "luke-warm", "pan-seared", "slow-cooked", "steamed", "peppered", "pickled", "fresh", "homemade", "yummy"]
+adj_two = ["acidic", "blazed", "burnt", "cheesy", "deep-fried", "encrusted", "fluffy", "frozen", "greesy", "infused", "marinated"]
+food_type = ["roast", "enchiladas", "pancakes", "pastry", "custard", "salad", "casserole", "pasta", "vegetables", "soup", "popsicle"]
+
+
+# Length of the arrays
+array_length = adj_one.length
```

Comment:
Since you only use this variable once - you probably don't need to use a local variable and instead just this `length` call on line 17